src/components/patient-banner/patient-banner.scss

```diff
@@ -5,12 +5,13 @@
     width: 100%;
     height: 5.5rem;
     position:fixed;
-    z-index: 8000;
+    margin-top: 48px;
+    // z-index: 8000;
     background: #ffffff;
-    margin: 0 0 1rem;
-    padding: 1rem 0.5rem 1rem;
-    //TODO: change from px to rem;
-    margin-top: -8px;
+    // margin: 0 0 1rem;
```

Comment:
@herbertUG delete all unused code rather than commenting it out

Reply:
@ssmusoke surething.